### PR TITLE
TF Lite Updated activations.cc

### PR DESCRIPTION
**Pull request by PariksheetPinjari909**
Corrected log error in activations.cc

### CODE REVIEW

File: tensorflow/lite/kernels/activations.cc [hunk 1]
```diff
@@ -473,8 +473,10 @@ TfLiteStatus TanhEval(TfLiteContext* context, TfLiteNode* node) {
       return kTfLiteOk;
     } break;
     default:
-      context->ReportError(context, "Only float32 supported currently, got %s.",
-                           TfLiteTypeGetName(input->type));
+      context->ReportError(
+          context,
+          "Only float32, uint8, int16 and int8 supported currently, got %s.",
```
Comment (renjie-liu) [inline tensorflow/lite/kernels/activations.cc]: nit: *are*

File: tensorflow/lite/kernels/activations.cc [hunk 2]
```diff
@@ -539,8 +541,11 @@ TfLiteStatus SigmoidEval(TfLiteContext* context, TfLiteNode* node) {
       break;
     }
     default:
-      context->ReportError(context, "Only float32 supported currently, got %s.",
-                           TfLiteTypeGetName(input->type));
+      context->ReportError(
+          context,
+          "Only float32, uint8, int16 and int8 supported currently, got %s.",
```
Comment (renjie-liu) [inline tensorflow/lite/kernels/activations.cc]: same here

File: tensorflow/lite/kernels/activations.cc [hunk 3]
```diff
@@ -876,8 +881,9 @@ TfLiteStatus LogSoftmaxEval(TfLiteContext* context, TfLiteNode* node) {
       return kTfLiteOk;
     }
     default:
-      context->ReportError(context, "Only float32 supported currently., got %s",
-                           TfLiteTypeGetName(input->type));
+      context->ReportError(
+          context, "Only float32, uint8 and int8 supported currently, got %s.",
```
Comment (renjie-liu) [inline tensorflow/lite/kernels/activations.cc]: and here

**Comment by PariksheetPinjari909** [comment]
@renjie-liu Thanks for the review comments. I have updated accordingly.

**Review by renjie-liu — APPROVED** [review]

**Pull request merged**